x-pack/plugins/security_solution/public/timelines/components/open_timeline/edit_timeline_batch_actions.tsx

```diff
@@ -106,14 +109,15 @@ export const useEditTimelineBatchActions = ({
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
do we still need `eslint-disable`?

Reply:
seems there is a bug in the eslint rule and because of that it's still required